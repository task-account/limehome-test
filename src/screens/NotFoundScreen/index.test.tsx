import renderer from "react-test-renderer";
import { NotFoundScreen } from ".";

jest.mock("@utils/hooks", () => ({
  withStyles: () => () => ({ useStyles: jest.fn() })
}));

describe(`NotFoundScreen`, () => {
  it("render", () => {
    const tree = renderer.create(<NotFoundScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
