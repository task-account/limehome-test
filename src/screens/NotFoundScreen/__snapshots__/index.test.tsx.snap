// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundScreen render 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <Text>
    Oops! Screen is missing
  </Text>
</RCTSafeAreaView>
`;
